ssr/src/consts/mod.rs

```diff
@@ -72,3 +72,19 @@ pub mod google {
     pub const GOOGLE_TOKEN_URL: &str = "https://oauth2.googleapis.com/token";
     pub const GOOGLE_ISSUER_URL: &str = "https://accounts.google.com";
 }
+
+#[derive(Clone)]
+pub struct HardCodedIDs {
+    pub ledger: &'static str,
+    pub index: &'static str,
+}
+pub static HARDCODED_TOKEN_IDS: phf::Map<&'static str, HardCodedIDs> = phf::phf_map! {
+    "btc" => HardCodedIDs {
+        ledger: "mxzaz-hqaaa-aaaar-qaada-cai",
+        index: "n5wcd-faaaa-aaaar-qaaea-cai",
+    },
+    "usdc" => HardCodedIDs {
+        ledger: "xevnm-gaaaa-aaaar-qafnq-cai",
+        index: "xrs4b-hiaaa-aaaar-qafoa-cai"
+    }
+};
```

Comment:
Is this something we can't do with a `match` block?

Reply:
Yes but we're are required to match it multiple times in the codebase to infer the param... Like for example we could be using an actual token root or we way just pass in btc or usdc as the param so we need to match this many times so it'd be better to have a static map and then get it whenever we need it... Or prolly could use a function instead...

Reply:
you can just use a function, the compiler will convert the code to a static lookup table internally (which is probably more efficient than  phf)